tests/functional/test_basic.py

```diff
@@ -39,6 +39,21 @@ def test_absolute_python_path(pyi_builder):
     pyi_builder.test_script('pyi_absolute_python_path.py')
 
 
+@skipif_winorosx
```

Comment:
Rethinking this: The test-case is linux-specific and not run on e.g. FreeBSD. Thus this should be
```suggestion
@skipif_notlinux
```
(I just added `@skipif_notlinux`, so you need to rebase your code.)

Reply:
Fixed myself.